app/src/main/java/com/github/multimatum_team/multimatum/activity/ProfileActivity.kt

```diff
@@ -34,8 +38,12 @@ class ProfileActivity : AppCompatActivity() {
 
         logOutButton.setOnClickListener {
             Log.d(TAG, "Logging out...")
-            userViewModel.signOut()
-            AuthUI.getInstance().signOut(this)
+            if((this.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager).activeNetwork != null) {
```

Comment:
Does this could be tested or you don't have the time ?

Reply:
It depends on weird case where we don't have internet connection and I'm not sure that we could "disable" internet connection in test and If it's possible search how it's works will take me too much time